client/extensions/woocommerce/state/wc-api/settings/general/reducer.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Internal dependencies
+ */
+import {
+	WOOCOMMERCE_API_FETCH_SETTINGS_GENERAL,
+	WOOCOMMERCE_API_FETCH_SETTINGS_GENERAL_SUCCESS,
+} from '../../../action-types';
+
+export const LOADING = 'LOADING';
```

Comment:
I am following the pattern used by @DanReyLop in the shipping reducer.  Can we get this and my other PR in and then bulk change them at the same time?